extension/src/setup/index.ts

```diff
@@ -310,6 +311,9 @@ export class Setup
     const root = getFirstWorkspaceFolder()
     if (root) {
       await this.dvcExecutor.init(root)
+      void Toast.infoWithOptions(
+        'Initialized DVC repository. You can now [commit the changes to git](command:workbench.view.scm).'
```

Comment:
[F] It is also a more expected pattern to make this a button by pasting `items` to `Toast.infoWithOptions`. See https://github.com/iterative/vscode-dvc/blob/09e32508d93c62914deac0084a2578a7710612e5/extension/src/cli/dvc/discovery.ts#L47-L48 for an example

Reply:
> [F] It is also a more expected pattern to make this a button by pasting items to Toast.infoWithOptions. 

Makes sense! Would say I'll update it in the followup, but now that I think about it, we will probably want to take off the Toast once we have the Setup screen 🤔